.github/workflows/dotnet-instr-deployer-add-on.yml

```diff
@@ -46,7 +60,8 @@ jobs:
 
       - name: Download the Splunk UF MSI
         shell: bash
-        run: curl https://download.splunk.com/products/universalforwarder/releases/9.4.0/windows/splunkforwarder-9.4.0-6b4ebe426ca6-windows-x64.msi -o splunkforwarder.msi
+        run: |
+          curl https://download.splunk.com/products/universalforwarder/releases/${{ env.splunk_uf_version }}/windows/splunkforwarder-${{ env.splunk_uf_version }}-${{ env.splunk_uf_build_hash }}-windows-x64.msi -o splunkforwarder.msi
```

Comment:
Do we know that the naming scheme of the msi here will always be consistent with version-buildhash?

Reply:
As far as I can tell yes, this schema goes back many versions and hasn't changed in years.